solutions/gold/usaco-950.mdx

```diff
@@ -2,13 +2,49 @@
 id: usaco-950
 source: USACO Platinum 2019 Open
 title: Valleys
-author: Benjamin Qi
+author: Benjamin Qi, Alexis Tao
 ---
 
+## Explanation
```

Comment:
add a focus problem

Reply:
Yeah, I can make it a focus problem but I think the Valleys problem also appears in Gold - Disjoint Set Union, should I delete the internal solution

Reply:
nononono
just add it as a focus problem here, i don't see why we have to delete it